### ISSUE TITLE
Direct interpretation emits invalid messages

### ISSUE THREAD

**Issue by timrobertson100** [gbif/occurrence#14]
When running a reprocess, we put messages in on the `direct-interp` queue.
However, the messages emitted at the end to e.g. the maps, seem to have `null` for all values.  The maps will then be mutated incorrectly.

[An example message pulled from a map queue](https://gist.github.com/timrobertson100/237d018d5da05339df6d8282714365a4) shows what happens when a record is interpreted once, and then interpreted again - the example shows the second interpretation. (This is repeated below in a comment case the gist disappears)

In an ideal scenario, no message would be emitted because the record should not change the second time.  If it were emitted with correct values that would also be acceptable though, as maps and cubes would detect no change.

**Comment by timrobertson100** [comment]
```JSON
{  
   "datasetUuid":"4fa7b334-ce0d-4e88-aaae-2e0c138d049e",
   "status":"UPDATED",
   "crawlAttempt":7,
   "oldOccurrence":{  
      "key":1161897615,
      "datasetKey":"4fa7b334-ce0d-4e88-aaae-2e0c138d049e",
      "publishingOrgKey":"e2e717bf-551a-4917-bdc9-4fa0f342c530",
      "publishingCountry":"US",
      "protocol":"DWC_ARCHIVE",
      "lastCrawled":1480007976552,
      "lastParsed":1480008326927,
      "crawlId":null,
      "extensions":{  

      },
      "basisOfRecord":null,
      "individualCount":null,
      "sex":null,
      "lifeStage":null,
      "establishmentMeans":null,
      "taxonKey":null,
      "kingdomKey":null,
      "phylumKey":null,
      "classKey":null,
      "orderKey":null,
      "familyKey":null,
      "genusKey":null,
      "subgenusKey":null,
      "speciesKey":null,
      "scientificName":null,
      "kingdom":null,
      "phylum":null,
      "order":null,
      "family":null,
      "genus":null,
      "subgenus":null,
      "species":null,
      "genericName":null,
      "specificEpithet":null,
      "infraspecificEpithet":null,
      "taxonRank":null,
      "dateIdentified":null,
      "decimalLongitude":null,
      "decimalLatitude":null,
      "coordinatePrecision":null,
      "coordinateUncertaintyInMeters":null,
      "coordinateAccuracy":null,
      "elevation":null,
      "elevationAccuracy":null,
      "depth":null,
      "depthAccuracy":null,
      "continent":null,
      "stateProvince":null,
      "waterBody":null,
      "year":null,
      "month":null,
      "day":null,
      "eventDate":null,
      "typeStatus":null,
      "typifiedName":null,
      "issues":[  

      ],
      "modified":null,
      "lastInterpreted":null,
      "references":null,
      "license":null,
      "identifiers":[  

      ],
      "media":[  

      ],
      "facts":[  

      ],
      "relations":[  

      ],
      "geodeticDatum":null,
      "class":null,
      "countryCode":null,
      "country":null,
      "http://rs.tdwg.org/dwc/terms/country":"United States",
      "http://rs.tdwg.org/dwc/terms/order":"Passeriformes",
      "county":"Monroe",
      "gbifID":"1161897615",
      "http://rs.tdwg.org/dwc/terms/year":"2014",
      "http://rs.tdwg.org/dwc/terms/basisOfRecord":"HumanObservation",
      "catalogNumber":"OBS242531339",
      "institutionCode":"CLO",
      "http://rs.tdwg.org/dwc/terms/family":"Cardinalidae",
      "http://rs.tdwg.org/dwc/terms/decimalLatitude":"43.3424553",
      "http://rs.tdwg.org/dwc/terms/month":"03",
      "http://rs.tdwg.org/dwc/terms/scientificName":"Cardinalis cardinalis",
      "http://rs.tdwg.org/dwc/terms/stateProvince":"New York",
      "http://rs.tdwg.org/dwc/terms/phylum":"Chordata",
      "identifier":"OBS242531339",
      "http://rs.tdwg.org/dwc/terms/day":"30",
      "http://rs.tdwg.org/dwc/terms/kingdom":"Animalia",
      "http://rs.tdwg.org/dwc/terms/genus":"Cardinalis",
      "http://rs.tdwg.org/dwc/terms/class":"Aves",
      "locality":"** Home (Hamlin)",
      "collectionCode":"EBIRD",
      "occurrenceID":"URN:catalog:CLO:EBIRD:OBS242531339",
      "http://rs.tdwg.org/dwc/terms/individualCount":"3",
      "http://rs.tdwg.org/dwc/terms/specificEpithet":"cardinalis",
      "recordedBy":"obsr95197",
      "http://rs.tdwg.org/dwc/terms/decimalLongitude":"-77.9539633",
      "http://rs.tdwg.org/dwc/terms/geodeticDatum":"WGS84"
   }   {  
      "key":1161897615,
      "datasetKey":"4fa7b334-ce0d-4e88-aaae-2e0c138d049e",
      "publishingOrgKey":"e2e717bf-551a-4917-bdc9-4fa0f342c530",
      "publishingCountry":"US",
      "protocol":"DWC_ARCHIVE",
      "lastCrawled":1480007976552,
      "lastParsed":1480008326927,
      "crawlId":null,
      "extensions":{  

      },
      "basisOfRecord":"HUMAN_OBSERVATION",
      "individualCount":3,
      "sex":null,
      "lifeStage":null,
      "establishmentMeans":null,
      "taxonKey":2490384,
      "kingdomKey":1,
      "phylumKey":44,
      "classKey":212,
      "orderKey":729,
      "familyKey":9285,
      "genusKey":3241527,
      "subgenusKey":null,
      "speciesKey":2490384,
      "scientificName":"Cardinalis cardinalis (Linnaeus, 1758)",
      "kingdom":"Animalia",
      "phylum":"Chordata",
      "order":"Passeriformes",
      "family":"Cardinalidae",
      "genus":"Cardinalis",
      "subgenus":null,
      "species":"Cardinalis cardinalis",
      "genericName":"Cardinalis",
      "specificEpithet":"cardinalis",
      "infraspecificEpithet":null,
      "taxonRank":"SPECIES",
      "dateIdentified":null,
      "decimalLongitude":-77.953963,
      "decimalLatitude":43.342455,
      "coordinatePrecision":null,
      "coordinateUncertaintyInMeters":null,
      "coordinateAccuracy":null,
      "elevation":null,
      "elevationAccuracy":null,
      "depth":null,
      "depthAccuracy":null,
      "continent":null,
      "stateProvince":"New York",
      "waterBody":null,
      "year":2014,
      "month":3,
      "day":30,
      "eventDate":1396137600000,
      "typeStatus":null,
      "typifiedName":null,
      "issues":[  
         "COORDINATE_ROUNDED"
      ],
      "modified":null,
      "lastInterpreted":1490514654355,
      "references":null,
      "license":"CC0_1_0",
      "identifiers":[  

      ],
      "media":[  

      ],
      "facts":[  

      ],
      "relations":[  

      ],
      "geodeticDatum":"WGS84",
      "class":"Aves",
      "countryCode":"US",
      "country":"United States",
      "http://rs.tdwg.org/dwc/terms/country":"United States",
      "http://rs.tdwg.org/dwc/terms/order":"Passeriformes",
      "county":"Monroe",
      "gbifID":"1161897615",
      "http://rs.tdwg.org/dwc/terms/year":"2014",
      "http://rs.tdwg.org/dwc/terms/basisOfRecord":"HumanObservation",
      "catalogNumber":"OBS242531339",
      "institutionCode":"CLO",
      "http://rs.tdwg.org/dwc/terms/family":"Cardinalidae",
      "http://rs.tdwg.org/dwc/terms/decimalLatitude":"43.3424553",
      "http://rs.tdwg.org/dwc/terms/month":"03",
      "http://rs.tdwg.org/dwc/terms/scientificName":"Cardinalis cardinalis",
      "http://rs.tdwg.org/dwc/terms/stateProvince":"New York",
      "http://rs.tdwg.org/dwc/terms/phylum":"Chordata",
      "identifier":"OBS242531339",
      "http://rs.tdwg.org/dwc/terms/day":"30",
      "http://rs.tdwg.org/dwc/terms/kingdom":"Animalia",
      "http://rs.tdwg.org/dwc/terms/genus":"Cardinalis",
      "http://rs.tdwg.org/dwc/terms/class":"Aves",
      "locality":"** Home (Hamlin)",
      "collectionCode":"EBIRD",
      "occurrenceID":"URN:catalog:CLO:EBIRD:OBS242531339",
      "http://rs.tdwg.org/dwc/terms/individualCount":"3",
      "http://rs.tdwg.org/dwc/terms/specificEpithet":"cardinalis",
      "recordedBy":"obsr95197",
      "http://rs.tdwg.org/dwc/terms/decimalLongitude":"-77.9539633",
      "http://rs.tdwg.org/dwc/terms/geodeticDatum":"WGS84"
   },
   "deletionReason":null,
   "crawlAttemptLastSeen":null,
   "latestCrawlAttemptForDataset":null
}
```

**Comment by timrobertson100** [comment]
This appears to happen with normal crawling too... not just the forced reinterpreting

**Comment by MattBlissett** [comment]
Fixed in 4beca9df3292c68aafef0a92ab8826b82f74f58e
